06_gpu_and_ml/comfyui/impact/impact_example.py

```diff
@@ -0,0 +1,44 @@
+# ---
+# cmd: ["modal", "serve", "06_gpu_and_ml/comfyui/impact/impact_example.py"]
+# ---
+
+import subprocess
+
+import modal
+
+image = (
+    modal.Image.debian_slim(  # start from basic Linux with Python
+        python_version="3.11"
+    )
+    .apt_install("git")  # install git to clone ComfyUI
+    .pip_install("comfy-cli==1.2.7")  # install comfy-cli
+    .run_commands(  # use comfy-cli to install the ComfyUI repo and its dependencies
+        "comfy --skip-prompt install --nvidia"
+    )
+    .run_commands(  # download the Impact pack
+        "comfy node install ComfyUI-Impact-Pack"
+    )
+    .pip_install("ultralytics==8.3.26")  # object detection models
+    .apt_install(  # opengl dependencies
+        "libgl1-mesa-glx", "libglib2.0-0"
+    )
```

Comment:
Nice, I think it's helpful to have this to show that for some of these custom nodes you need to install other dependencies as well